src/ServiceManager/Travely.ServiceManager.DAL/Repositories/Repository.cs

```diff
@@ -37,14 +37,15 @@ public virtual async Task<IEnumerable<TEntity>> GetAllAsync()
             return await _dbSet.ToListAsync();
         }
 
-        public virtual async Task<TEntity> GetAsync(int entityId)
+        public virtual async Task<TEntity> GetAsync(long entityId)
         {
             return await _dbSet.FindAsync(entityId);
         }
 
         public virtual void Update(TEntity entity)
         {
             _dbSet.Update(entity);
+            _dbContext.SaveChanges();
```

Comment:
Do we really want to have SaveChanges() called in Repository if we have UnitOfWork?